lib/Migrator/APIDiffMigratorPass.cpp

```diff
@@ -1305,6 +1395,14 @@ struct APIDiffMigratorPass : public ASTMigratorPass, public SourceEntityWalker {
             handleLocalParameterBridge(AFD, DiffItem);
         }
       }
+      auto OverrideLoc = shouldRemoveOverride(AFD);
+      if (OverrideLoc.isValid()) {
+        // Remove override keyword.
+        Editor.remove(OverrideLoc);
+        // Remove super-dot call.
+        SuperRemoval Removal(Editor);
```

Comment:
This doesn't seem like the right place for this one. E.g. I might also use super.changeColor() in methods other than my override of changeColor() if I don't want to use my override.